recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pymultinest
+  version: "2.6"
+
+source:
+  fn: pymultinest-2.6.tar.gz
+  url: https://pypi.python.org/packages/c2/fc/77bdd8356068125bba0a5c07bfa2b4a3f492bff614aeae783bc1a566d699/pymultinest-2.6.tar.gz
```

Comment:
Please use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

once you adopt templating variables.

Reply:
done